netket/models/fast_autoreg.py

```diff
@@ -42,18 +42,25 @@ def conditional(self, inputs: Array, index: int) -> Array:
         if inputs.ndim == 1:
```

Comment:
I know this is already here, but could you add a better explanation to the docstring of this module explaining what the fast is for, and when to expect a speedup with this kind of implementation over the standard one?

Reply:
Previously all docs about fast AR sampling show the reference to `FastMaskedConv1D`, and I agree that it looks better to put that in `FastARNNSequential`